packages/form-layout/src/vaadin-form-layout.js

```diff
@@ -329,7 +329,7 @@ class FormLayout extends ResizeMixin(ElementMixin(ThemableMixin(PolymerElement))
   /** @private */
   _isValidCSSLength(value) {
     // Let us choose a CSS property for validating CSS <length> values:
-    // - `border-spacing` accepts `<length> | inherit`, it’s the best! But
+    // - `border-spacing` accepts `<length> | inherit`, it's the best! But
```

Comment:
Note: we might revisit the logic here as MSIE support was dropped long ago. But let's do that later.